content/objects/world/proc/odhotreloadresource.md

```diff
@@ -4,4 +4,9 @@ title = "ODHotReloadResource"
 is_override = false
 [[extra.args]]
 name = "file_name"
-+++
\ No newline at end of file
+type = "text"
```

Comment:
```suggestion
type_override = "text"
```
i just realised that this would be crushed by the autogen tool next time it runs, so it can be put in type_override instead https://github.com/OpenDreamProject/od-dm-reference/pull/23

but on that note, should all the fields pulled from dmstandard be marked, like `dmstd_name` and `dmstd_type`, and the human edited fields just be `name` and `type` and take priority in rendering?

Reply:
I'd definitely say human edited field should take priority over auto-gen'd ones.